components/storage/blockdevice/COMPONENT_OSPIF/mbed_lib.json

```diff
@@ -0,0 +1,44 @@
+{
+"name": "ospif",
+    "config": {
+        "OSPI_IO0": "MBED_CONF_DRIVERS_OSPI_IO0",
+        "OSPI_IO1": "MBED_CONF_DRIVERS_OSPI_IO1",
+        "OSPI_IO2": "MBED_CONF_DRIVERS_OSPI_IO2",
+        "OSPI_IO3": "MBED_CONF_DRIVERS_OSPI_IO3",
+        "OSPI_IO4": "MBED_CONF_DRIVERS_OSPI_IO4",
+        "OSPI_IO5": "MBED_CONF_DRIVERS_OSPI_IO5",
+        "OSPI_IO6": "MBED_CONF_DRIVERS_OSPI_IO6",
+        "OSPI_IO7": "MBED_CONF_DRIVERS_OSPI_IO7",
+        "OSPI_SCK": "MBED_CONF_DRIVERS_OSPI_SCK",
+        "OSPI_CSN": "MBED_CONF_DRIVERS_OSPI_CSN",
+	"OSPI_DQS": "MBED_CONF_DRIVERS_OSPI_DQS",
+        "OSPI_POLARITY_MODE": 0,
+        "OSPI_FREQ": "2000000",
```

Comment:
Set default frequency to your frequency ?

Reply:
default frequency